pages/manual/planet/course.md

```diff
@@ -16,11 +16,11 @@ Topics:
 ## Getting to the Courses Page
 There are two ways to get to the Courses page:
 
-1. From the Navigation Bar, click on the "Courses" Tab at the top.
+1.[Option 1] From the Navigation Bar, click on the "Courses" Tab at the top.
 
 ![Getting to courses with Navigation Bar](images/get-courses-navbar.png)
 
-2. From the dashboard tile, click on the "myCourses" tile right on the front page.
+2.[Option 2] From the dashboard tile, click on the "myCourses" tile right on the front page.
```

Comment:
I think you should just put the 1. and 2. (Because in the screen shot you provided there is a repetition of number so just fixing that would be enough.

Reply:
@aasenomad you should fix it to be numbering. Other changes are not required.
Also can you create clean PR instead of creating PR from old branch. Looks like you are not updating your fork repository cleanly, instead merging your own branches